pkg/kv/kvserver/replica_closedts_internal_test.go

```diff
@@ -565,10 +565,12 @@ func TestReplicaClosedTimestamp(t *testing.T) {
 			cfg.TestingKnobs.DontCloseTimestamps = true
 			cfg.ClosedTimestampReceiver = &r
 			tc.StartWithStoreConfig(ctx, t, stopper, cfg)
+			tc.repl.raftMu.Lock()
```

Comment:
Hmm, in light of the need to hold both mutexes, `mu.orRaftMu` is a deceptive name. (Disregard if this changes by the end of the commit list).